docs/pages/admin-guides/access-controls/rbac/getting-started-rbac.mdx

```diff
@@ -0,0 +1,315 @@
+and are used for internal users (created directly in Teleport) and external users (authenticated via Single Sign-On, like Okta or GitHub).
+Users can be assigned one or more roles, and the permissions of multiple roles combine to form a user’s overall capabilities.
+
+Admins create and assign roles to manage who can access which parts of the infrastructure.
+
+Some examples of roles include:
+
+- **access**: Grants basic access to resources.
+- **editor**: Allows users to modify configuration settings.
+- **auditor**: Allows users to view logs and auditing activities.
+
+Roles enforce Role-Based Access Control (RBAC), ensuring that users only have the permissions needed for their specific tasks. 
+See [Teleport Access Controls Reference](../../../reference/access-controls/roles.mdx) for in-depth information about types of roles, options, and usage.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+(!docs/pages/includes/permission-warning.mdx!)
+
+## Step 1/3. Add local users with preset roles
+
+Teleport provides several preset roles:
+
+(!docs/pages/includes/preset-roles-table.mdx!)
+
+<Tabs>
+<TabItem scope={["Teleport Community Edition"]} label="Teleport Community Edition">
+Invite the local user Alice as cluster `editor`:
+
+```code
+$ tctl users add alice --roles=editor
+```
+</TabItem>
+<TabItem scope={["enterprise", "cloud"]} label="Commercial">
+Invite the local user Alice as cluster `editor` and `reviewer`:
+
+```code
+$ tctl users add alice --roles=editor,reviewer
+```
+</TabItem>
+
+</Tabs>
+
+Once Alice signs up, she will be able to edit cluster configuration. You can list
+users and their roles using `tctl users ls`.
+
+<Tabs>
+<TabItem scope={"Teleport Community Edition"} label="Teleport Community Edition">
+```code
+$ tctl users ls
+
+# User                 Roles
+# -------------------- --------------
+# alice                editor
+```
+</TabItem>
+<TabItem scope={["enterprise", "cloud"]} label="Commercial">
+```code
+$ tctl users ls
+
+# User                 Roles
+# -------------------- --------------
+# alice                editor, reviewer
+```
+</TabItem>
+
+</Tabs>
+
+You can update the user's roles using the `tctl users update` command:
+
+<Tabs>
+<TabItem scope={"Teleport Community Edition"} label="Teleport Community Edition">
+```code
+# Once Alice logs back in, she will be able to view audit logs
+$ tctl users update alice --set-roles=editor,auditor
+```
+</TabItem>
+<TabItem scope={["enterprise", "cloud"]} label="Commercial">
+```code
+# Once Alice logs back in, she will be able to view audit logs
+$ tctl users update alice --set-roles=editor,reviewer,auditor
+```
+</TabItem>
+
+</Tabs>
+
+Because Alice has two or more roles, permissions from those roles create a union. She
+will be able to act as a system administrator and auditor at the same time.
+
+## Step 2/3. Map SSO users to roles
```

Comment:
We have a guide for early-stage users to get started with SSO (https://goteleport.com/docs/admin-guides/access-controls/sso/). I would refer readers to that page instead of attempting to explain the same material in a short section.

Reply:
That makes sense, removed